backend/program/media.py

```diff
@@ -276,7 +281,7 @@ def __iadd__(self, other):
         return self
 
     def sort(self, by, reverse):
-        self.items.sort(key=lambda item: item.get(by), reverse=reverse)
+        self.items.sort(key=lambda item: item.get(by) if item is not None else None)

```

Comment:
This will break stuff, we cant have None items anyway so you are doing something wrong

Reply:
Fixed